samples/meetings-token-app/csharp/Startup.cs

```diff
@@ -47,10 +47,10 @@ public void ConfigureServices(IServiceCollection services)
             services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme).AddJwtBearer(options =>
             {
                 options.Authority = "https://login.microsoftonline.com/common";
-                options.Audience = this.Configuration["AzureAd:ApplicationIdURI"];
```

Comment:
Was this removed to support multi-tenant scenarios?

Reply:
Yes